website/assets/sass/_searchview.scss

```diff
@@ -49,6 +49,33 @@ $scroll-max-height: $results-view-max-height - 46;
         }
     }
 
+    .icon-bar {
```

Comment:
Is this class used anywhere? I couldn't find it in this pull request

Reply:
is not.! I forgot to erase that.